index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
+        <header class="header">
+            
+            <nav class="nav">
+                <a href="#logo" class="logo">John Doe</a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
```

Comment:
- [ ] It was requested by the previous to change the generic `div` tag to  `nav` tag for semantics reason. But you mistakenly changed different div. Below is how your `header` section should look like.

```
    <header class="header">
         <a href="#logo" class="logo">John Doe</a>       
         <nav class="hamburger">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
         </nav>
    <header>
```

Reply:
Fixed.

Reply:
hi @Abenezer-Tilahun how do i fix the mit license? kindly help